spacepy/time.py

```diff
@@ -2249,6 +2249,36 @@ def valid_YYYYMMDD(inval):
         return False
 
 
+def _leapsgood(now, filetime, lastleap):
+    """Determines if leap seconds are up-to-date
+
+    Parameters
+    ----------
+    now : datetime.datetime
+        Current time (UTC assumed)
+    filetime : datetime.datetime
+        Timestamp (mtime) of the leapsecond file
+    lastleap : datetime.datetime
+        Last leapsecond in the file (really the moment after the leap)
+
+    Returns
+    -------
+    bool
+        True if leap second file is up-to-date, False if might not be.
+    """
+    # There cannot be a leap second until AFTER the ls after the
+    # bulletin (e.g. once pass 1 Jan, know the next possible is next 1 Jan)
```

Comment:
Shouldn't this be "... once past 1 Jan ..."

Reply:
I guess it's a code comment and not docs, so it probably doesn't matter - I think I initially assumed it was docstring. Tired eyes, etc.

Reply:
What I meant once "once we pass 1 Jan" but I'm happy to change tense; I do think it's clearer. I'll do that a sec and ping you once the tests are done.